### DISCUSSION TITLE
how to install it on PC (no response after clicking the setup.exe)

### DISCUSSION THREAD

**Discussion by schung168** [audiamus/AaxAudioConverter#165] [Q&A]
I have downloaded AaxAudioConverter-1.18-Setup.exe on Windows11.  I don't see any response after I click on the setup.exe (either with Open or Run As Administrator).  Is there any hidden issue with this program?  How can I work around to get it working?
Thanks!

**Comment by audiamus** [comment]
It's a standard InnoSetup package, nothing special about it. Your download may be broken. Try to re-download or check the MD5 hash:
https://github.com/audiamus/AaxAudioConverter/blob/master/res/Setup.dat
Is 1.18 the first AAX Audio Converter release you try to install? If you are upgrading from a previous release, the auto update would check the integrity of the download before launching the setup.

**Reply by schung168** [reply]
**The current PC is running windows 11:**
1.18 is the first version I tried with AAX Audio Converter.
I downloaded it again.  Checked MD5 and it matched.  
But the same problem remains.  After I double click the program from explorer, nothing come up.

I also checked the earlier download that had the same issue. Its MD5 record matches to what you provided as well.

I used another older PC with windows 10.  The installation works.  Does this program AAX Audio Converter not work with windows 11?

**Reply by audiamus** [reply]
Since the installation package is not digitally signed there could be some sort of security setting in Windows to prevent the installer to start. But the Windows 11 default settings should allow the installation nonetheless. Does the Windows event log have any entries related to this?

**Reply by schung168** [reply]
I am looking at "Event Viewer".  Which part of the data should I search for this entry?

**Comment by schung168** [comment]
I am looking at "Event Viewer". Which part of the data should I search for
this entry?

On Sat, Jan 21, 2023 at 1:11 PM audiamus ***@***.***> wrote:

> Since the installation package is not digitally signed there could be some
> sort of security setting in Windows to prevent the installer to start. But
> the Windows 11 default settings should allow the installation nonetheless.
> Does the Windows event log have any entries related to this?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/audiamus/AaxAudioConverter/discussions/165#discussioncomment-4746146>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABMXIKQQEIFVG63BSPG6J63WTQRFVANCNFSM6AAAAAAUAZDZTE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Stephen Chung

**Reply by audiamus** [reply]
I can't really say. I'd guess _Security_ or _Application_ in _Windows Logs_. There may be clues there, there may be not.

You can try an older version of AAX Audio Converter or its companion Book Lib Connect. If none of these installers run then it's probably some setting on your machine. But I don't know what setting that would be.

**Reply by schung168** [reply]
where can I find older version of AAX Audio Converter and Book Lib Connect?

**Reply by audiamus** [reply]
https://github.com/audiamus/AaxAudioConverter/releases
https://github.com/audiamus/BookLibConnect/releases

They all use InnoSetup as the installer.

**Comment by schung168** [comment]
1. Very interesting.  It works with installing 1.17 and then the 1.18
version was then downloaded and was installed.  Strange though.
2. I tried to convert a few books I currently have with audible.com.  It
worked but some tracks missing "#", i.e. I could see #1, #3, #4, etc but
could not see "#2" associated with the 2nd track.  If I play these mp3
files on my cell phone's mp3 player, will they follow the sequence?
3. In addition, I have some audiobooks from older accounts with audible.com,
which are not available with audible.com anymore.  Can they be converted
into mp3 with your tool?
4. After I am comfortable with your application, can I donate some money to
you for using your free tools?  And how can I do that?

Thanks,
Stephen


On Sun, Jan 22, 2023 at 3:04 PM audiamus ***@***.***> wrote:

> https://github.com/audiamus/AaxAudioConverter/releases
> https://github.com/audiamus/BookLibConnect/releases
>
> They all use InnoSetup as the installer.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/audiamus/AaxAudioConverter/discussions/165#discussioncomment-4751182>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABMXIKVABQDCQPMSDK3KH2DWTWHFDANCNFSM6AAAAAAUAZDZTE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Stephen Chung

**Reply by audiamus** [reply]
ad 1: Since all the setup packages are created by the same install utility this is strange indeed.
ad 2: The entire splitting functionality is driven by available input and quite a number of user settings. Input is either embedded chapter data or accompanying json chapter data and also silence detection, if splitting is set for below chapter level. And user settings also influence folder structure and file names. So there are many variables in the process. However, if more than a single output file is produced for a book, an additional .m3u playlist will be created which will be recognized by many players. The user manual has many more details.
ad 3: Basically yes, but you will need the activation bytes for those. If these older Audible accounts have not been deleted, you should be able to connect to them with legacy Audible Manager. If Audible Manger can be activated for such an account, it will download the activation bytes to your machine and AAX Audible Converter will find them.    
ad 4: AAX Audio Converter and Book Lib Connect was and is intended to be totally free software. The best support you can give is to spread the word.
